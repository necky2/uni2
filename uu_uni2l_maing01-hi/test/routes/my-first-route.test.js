import UU5 from "uu5g04";
import MyFirstRoute from "../../src/routes/my-first-route";

const { shallow } = UU5.Test.Tools;

describe(`UuUni2l.Routes.MyFirstRoute`, () => {
  it(`default props`, () => {
    const wrapper = shallow(<MyFirstRoute />);
    expect(wrapper).toMatchSnapshot();
  });
});
