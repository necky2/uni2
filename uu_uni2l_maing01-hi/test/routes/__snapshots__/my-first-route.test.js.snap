// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UuUni2l.Routes.MyFirstRoute default props 1`] = `
<div
  className="uuunil-routes-0"
>
  <UuUni2l.Bricks.AddFive />
  <UuUni2l.Bricks.AddFive
    number={3}
  />
  <UuUni2l.Bricks.AddFive
    number={300}
  />
</div>
`;
